pyvista/plotting/plotting.py

```diff
@@ -2245,6 +2246,11 @@ def add_composite(
             have these updates rendered, e.g. by changing the active scalars or
             through an interactive widget.  Defaults to ``False``.
 
+        lookup_table : pyvista.LookupTable, optional
+            Use a custom lookup table instead of a color map. If this is set,
+            all parameters controlling the color map like ``cmap`` and
+            ``n_colors`` will be ignored.
```

Comment:
Personally, I'd prefer to pass the `LookupTable` object as the `cmap` parameter. Is this supported? (lots of code to sift through)

Reply:
I can do that. I feel we have too many keyword arguments.